src/managers/SessionManager.ts

```diff
@@ -78,15 +78,23 @@ class SessionManager {
             sessionId: sessionId,
             playerName: PlayerManager.generateName(),
         } as JoinDTO;
-        const response: axios.AxiosResponse<UUID> = await api.post(
-            "/join",
-            requestBody,
-        );
-        if (!response.data) {
+        try {
+            const response: axios.AxiosResponse<UUID> = await api.post(
+                "/join",
+                requestBody,
+            );
+            PlayerManager.saveId(response.data);
+            location.pathname = "";
+        } catch (error) {
+            if (axios.isAxiosError(error) && error.response?.status) {
+                alert(error.response?.data.message);
```

Comment:
would suggest to only do playermanager delete here

Reply:
Because we will discuss next week about the prompts

Reply:
Without displaying any error message?
I think its confusing to a user why he cannot join a game without displaying some error
alert() is the easiest way i can think of